README.md

```diff
@@ -139,6 +139,8 @@ This module requires [camptocamp/systemd](https://forge.puppet.com/camptocamp/sy
 
 When managing the repo, it either needs [puppetlabs/apt](https://forge.puppet.com/puppetlabs/apt) or [puppet/epel](https://forge.puppet.com/puppet/epel).
 
+For ulimit support on non-systemd it depends on [herculesteam/augeasproviders_sysctl](https://forge.puppet.com/herculesteam/augeasproviders_sysctl).
```

Comment:
This should let the user know which versions are supported. Suggest a reference to the `.fixtures.yml` file to find out the versions that are tested with this module. That way when those numbers get bumped in the future, the README does not also have to be updated.

Reply:
That's kind of tricky. In fixtures we only test with master. I'm not sure what a good solution is. Having it in README is pretty much guaranteed to go out of date so I'm leaning to keeping it for the user. Given we only use 3 properties (name, ensure and value) I think it's safe to assume the API will continue to be the same.